modules/@apostrophecms/schema/ui/apos/components/AposInputBoolean.vue

```diff
@@ -9,7 +9,7 @@
           class="apos-sr-only"
           type="checkbox" :id="`${uid}-true`"
           :value="true" @change="setValue(true)"
-          :checked="value.data === true"
+          :checked="value && value.data === true"
```

Comment:
This kind of thing is going to wind up all over the components. Let's not do that. Instead the component that owns the AposSchema should always make a document using `field.def` as appropriate before handing it to `AposSchema`.

Reply:
If you look at the 2.x code this is how it's done, there's a browser-side newInstance method.